float64 roll
float64 pitch
float64 yaw
float64 leg_r
float64 leg_l
float64 arm_r
float64 arm_l
float64 force_avg
float64 sigma


